### ISSUE TITLE
Monet error

### ISSUE THREAD

**Issue by seedpress** [mymatuda/GnuspeechOnWindows#1]
When trying to run Monet, the following error message is thrown: This application failed to start because it could not find or load the Qt platform plugin "windows". Reinstalling the application may fix this problem.

I'm running Win 7 Pro, 64 bit.  Qt is not installed, but I thought the three Qt dll's you put in /bin would suffice. Should I just go ahead and install Qt? (It's a big download!)

Thanks for all your effort in porting Gnuspeech.

**Comment by mymatuda** [comment]
Thanks for testing. Please download the new package.

**Comment by mymatuda** [comment]
I'm sorry, I added the wrong libraries. Now I am using a clean VM to test.

**Comment by mymatuda** [comment]
There is a new package, it should work now.

**Comment by seedpress** [comment]
Thanks. I'll try the newest one out sometime today; however, the one you
put up yesterday, gnuspeech-bin-win32-r20150807, seemed to be working. I
did have trouble with "simpletract", but that might be some problem here
with Jack, which always seems to be unreliable.

On Sat, Aug 8, 2015 at 11:15 AM, Marcelo Y. Matuda <notifications@github.com

**Comment by seedpress** [comment]
Mr. Matuda,

Thanks again for the updates release. Both "simple_tract" and "monet_cp"
seem to be working well now. Of course, it would take some time to try out
all the features of Monet. My problem with Jack was due to my own faulty
configuration.

As long as I've got your attention, I wonder if you are also planning to
port the Gnuspeech application "Synthesizer" to Windows?

On another note, I've noticed that Professor Hill's Gnuspeech pages haven't
been updated in over three years, and so contain no mention of your most
recent work, nor of Steve Nygard's.

I appreciate all your efforts.

Cheers,
Bob Hamilton

On Sat, Aug 8, 2015 at 11:15 AM, Marcelo Y. Matuda <notifications@github.com
